sdcm/cluster.py

```diff
@@ -411,6 +412,10 @@ def short_hostname(self):
                 return "no_booted_yet"
         return self._short_hostname
 
+    @property
+    def ssh_login_info(self):
+        return self._ssh_login_info
```

Comment:
I think it would be better to replace all `_ssh_login_info` to `ssh_login_info`

Reply:
done